src/pymovements/dataset/dataset_files.py

```diff
@@ -207,20 +207,78 @@ def load_gaze_files(
                 extension=extension,
             )
 
-        gaze_df = load_gaze_file(
+        gaze_data = load_gaze_file(
             filepath=filepath,
             preprocessed=preprocessed,
             custom_read_kwargs=definition.custom_read_kwargs,
         )
 
         # Add fileinfo columns to dataframe.
-        gaze_df = add_fileinfo(
+        gaze_data = add_fileinfo(
             definition=definition,
-            df=gaze_df,
+            df=gaze_data,
             fileinfo=fileinfo_row,
         )
 
-        gaze_dfs.append(GazeDataFrame(gaze_df, experiment=definition.experiment))
+        if preprocessed and extension == 'feather':
+            # Preprocessed data already has tuple columns.
+            gaze_df = GazeDataFrame(
+                gaze_data,
+                experiment=definition.experiment,
+            )
+
+        elif preprocessed and extension == 'csv':
+            # Workaround for preprocessed column names
+            time_column = None
+            if 'time' in gaze_data.columns:
+                time_column = 'time'
+
+            pixel_columns: list[str] = list(
+                set(GazeDataFrame.valid_pixel_position_columns) & set(gaze_data.columns),
+            )
+            if len(pixel_columns) == 0:
```

Comment:
this and all following == 0 could also be `if not len(pixel_columns):`

Reply:
I didn't care that much about that section as this will have to be reworked somehow in future PRs for sure

I still replaced these if conditions with the even shorter but equivalent `if not pixel_columns:`